aws/aws-how-to/instances/data-science-stack-on-ec2.rst

```diff
@@ -0,0 +1,144 @@
+Canonical Data Science Stack on EC2 using a GPU-enabled instance type
+=======================================================================
+
+
+Canonical Data Science Stack is a command line interface-based tool that bundles Jupyter Notebooks, MLflow and frameworks like PyTorch and TensorFlow on top of an orchestration layer, making this product excellent for rapid testing, prototyping and doing ML at a small scale. 
+
+While this is a product intended for desktop machine learning users, you can also deploy it on EC2 following these instructions.
+
+We are using a G4DN instance type to leverage the GPU, required for machine learning training and inference.
+
+Learn more about Canonical Data Science Stack in our `press release post`_.
+
+Launch a GPU-EC2 instance (G4DN instance family)
+------------------------------------------------
+
+
+Navigate to the EC2 Web console, select Launch Instance and make sure you select either Ubuntu 22.04 or 24.04 LTS (free or Pro), and any G4DN instance type family.
+
+For this example, we are using 22.04 on ``g4dn.xlarge``, which has 4 vCPUs and 16 GiB of RAM. It is powered with an NVIDIA T4 GPU.
+
+Make a note of the machine IP and the Key-Pair used. You’ll need it for connecting to the machine.
+
+Log in and install GPU drivers
```

Comment:
should we reference the DSS docs here too?

Reply:
agreed